src/routes/lobby/components/LobbyPlayerIndicator.vue

```diff
@@ -111,6 +112,7 @@ export default {
   justify-content: center;
   align-items: center;
   border-radius: 8px;
+  border: 1px solid red;
```

Comment:
Is this just for testing? What is the purpose of this css?

Reply:
oups, yeah just for testing.